backend/src/test/kotlin/blue/mild/covid/vaxx/utils/LoginApiTestHelper.kt

```diff
@@ -0,0 +1,40 @@
+package blue.mild.covid.vaxx.utils
+
+import blue.mild.covid.vaxx.routes.Routes
+import com.fasterxml.jackson.databind.JsonNode
+import io.ktor.client.call.receive
+import io.ktor.client.request.accept
+import io.ktor.client.request.post
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import mu.KLogging
+import org.apache.http.HttpStatus
+
+
+abstract class LoginApiTestHelper(
+   targetHost: String,
+   requestTimeoutsSeconds: Int
+) : ApiCallTestHelper(targetHost, requestTimeoutsSeconds) {
+    private companion object : KLogging()
+
+    protected suspend fun runPatientLogin(
+        loginUserForTest: Any,
+        httpStatus: Int = HttpStatus.SC_OK,
```

Comment:
proc tady beres `Int`? imo by bylo lepsi primo brat ten datovej typ, kterej vraci `request.status` coz je nejakej Ktor Http Status nebo tak nejak

navic tenhle Apache HttpStatus je tu tak trochu "omylem" - mysleno jako ze ho nemame primo pridanej jako zavislost, ale je to tranzitivni z nejakeho jineho balicku